e2e/app/component/card-base.ts

```diff
@@ -1,26 +1,36 @@
 import {ElementHandle, Page} from 'puppeteer';
-import {EllipsisMenuAction} from 'app/text-labels';
-import EllipsisMenu from './ellipsis-menu';
+import {Option} from 'app/text-labels';
+import Container from 'app/container';
+import SnowmanMenu from './snowman-menu';
 
 
-export default abstract class CardBase {
+export default abstract class CardBase extends Container {
 
-  ellipsisXpath = './/clr-icon[@shape="ellipsis-vertical"]';
+  private snowmanIconXpath = './/clr-icon[@shape="ellipsis-vertical"]';
```

Comment:
How come this string is duplicated everywhere instead of living in `snowman-menu.ts`?

Reply:
Finding icon and clicking icon to trigger snowman menu to open is not in `snowman-menu` class because icons' xpath selectors are no same on all pages.

Reply:
Right, but for the most part, the `.//clr-icon[@shape="ellipsis-vertical"]` is used in all the xpath selectors to get the snowman icon. That text doesn't need to be duplicated everywhere.

Reply:
Where is a good place to put it as common str? Not so many places, I found only two files have the string.
![Screen Shot 2020-09-30 at 3 45 54 PM](https://user-images.githubusercontent.com/35533885/94732473-50942180-0334-11eb-9732-5521202c2486.png)

Reply:
I'd put it in `snowman-menu.ts`, export the const, then access that from `card-base.ts` and `cohort-participants-group.ts`

Reply:
yes, good call. Done!